arbeitszeit_web/www/presenters/remove_worker_from_company_presenter.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases.remove_worker_from_company import Response as UseCaseResponse
+from arbeitszeit_web.notification import Notifier
+from arbeitszeit_web.translator import Translator
+from arbeitszeit_web.url_index import UrlIndex
+from arbeitszeit_web.www.response import Redirect
+
+
+@dataclass
+class ErrorCode:
```

Comment:
do you really need a class for this single int? how about using a type alias instead?

Reply:
Actually I prefer it to be a dataclass... I guess implementing it as a type alias would be like that:

```
ErrorCode = int

ViewModel = Redirect | ErrorCode
```

The small problem I have with this: When I call the presenter method and get the ViewModel, I can use `view_model.url` if ViewModel is a `Redirect` instance, but I cannot use `view_model.code` if it is a `ErrorCode` instance.  Don't really know if it is a convincing reason to reject your proposel ;-)

Btw, note that also the `Redirect` class is a dataclass with a single field:

```
@dataclass
class Redirect:
    url: str
```

Reply:
You are right, it is just as type safe. I prefered the dataclass because it is similar to the Redirect class. If you think i should change it let me know.